tests/integration_tests/db_engine_specs/base_engine_spec_tests.py

```diff
@@ -229,11 +229,11 @@ def test_get_table_names(self):
 
         """ Make sure base engine spec removes schema name from table name
         ie. when try_remove_schema_from_table_name == True. """
-        base_result_expected = ["table", "table_2"]
+        base_result_expected = {"table", "table_2"}
         base_result = BaseEngineSpec.get_table_names(
             database=mock.ANY, schema="schema", inspector=inspector
         )
-        self.assertListEqual(base_result_expected, base_result)
+        self.assertSetEqual(base_result_expected, base_result)
```

Comment:
could we just do this as that's what appears to be preferred these days?
```
assert base_result_expected == base_result
```

Reply:
I agree. I didn't know if unittest supported this.